Paws4Me/Paws4Me/Display Single Pet Details/View Controllers/PetSingleDetailsViewController.swift

```diff
@@ -104,8 +73,26 @@ class PetSingleDetailsViewController: UIViewController {
 
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         if let destination = segue.destination as? LocalPetViewController {
-            destination.setNamePet(name: namePet)
-            destination.setImagePet(image: imgPet)
+            guard let petName = singlePetViewModel.singlePetName else { return }
+            guard let petImage = singlePetViewModel.singlePetImage else { return }
+            destination.setNamePet(name: petName)
+            destination.setImagePet(image: petImage)
+        }
+    }
+
+    private func isPetSaved(petName: String) {
+        do {
+            guard let pets = try viewContext?.fetch(Pet.fetchRequest()) else { return }
```

Comment:
Code like this belongs in the `repository`.

Reply:
Thank you